data/strings/strings.txt

```diff
@@ -15968,50 +15968,98 @@
+    he = OpenStreetMap.org (OSM) הוא פרויקט קהילתי לבניית מפה חופשית ופתוחה. זהו המקור העיקרי לנתוני מפות במפות אורגניות ופועל בדומה לוויקיפדיה. אתה יכול להוסיף או לערוך מקומות והם הופכים לזמינים למיליוני משתמשים בכל רחבי העולם.\nהצטרף לקהילה ועזור ליצור מפה טובה יותר לכולם!
+    hi = OpenStreetMap.org (OSM) एक निःशुल्क और खुला मानचित्र बनाने के लिए एक सामुदायिक परियोजना है। यह ऑर्गेनिक मैप्स में मानचित्र डेटा का मुख्य स्रोत है और विकिपीडिया के समान काम करता है। आप स्थानों को जोड़ या संपादित कर सकते हैं और वे दुनिया भर में लाखों उपयोगकर्ताओं के लिए उपलब्ध हो जाते हैं।\nसमुदाय में शामिल हों और सभी के लिए एक बेहतर मानचित्र बनाने में सहायता करें!
+    hu = Az OpenStreetMap.org (OSM) egy közösségi projekt, amelynek célja egy ingyenes és nyílt térkép létrehozása. Ez az Organic Maps térképadatok fő forrása, és a Wikipédiához hasonlóan működik. Helyeket adhatsz hozzá vagy szerkeszthetsz, és azok világszerte felhasználók milliói számára válnak elérhetővé.\nLépj be a közösségbe, és segíts egy jobb térképet készíteni mindenki számára!
+    id = OpenStreetMap.org (OSM) adalah sebuah proyek komunitas untuk membuat sebuah peta yang bebas dan terbuka. OSM merupakan sumber utama data peta di Organic Maps dan berfungsi mirip dengan Wikipedia. Anda dapat menambahkan atau mengedit tempat dan tempat tersebut tersedia untuk jutaan pengguna di seluruh Dunia.\nBergabunglah dengan komunitas dan bantu membuat peta yang lebih baik untuk semua orang!
+    it = OpenStreetMap.org (OSM) è un progetto comunitario per costruire una mappa libera e aperta. È la principale fonte di dati cartografici di Organic Maps e funziona in modo simile a Wikipedia. Può aggiungere o modificare luoghi e questi diventano disponibili per milioni di utenti in tutto il mondo.\nEntri a far parte della comunità e contribuisca a creare una mappa migliore per tutti!
+    ja = OpenStreetMap.org(OSM)は、フリーでオープンな地図を構築するコミュニティ・プロジェクトです。OSMは、Organic Mapsの地図データの主要なソースであり、ウィキペディアと同じように機能します。あなたが場所を追加したり編集したりすると、世界中の何百万人ものユーザーがその場所を利用できるようになります！
+    ko = 오픈스트리트맵(OSM)은 무료 오픈스트리트맵을 구축하기 위한 커뮤니티 프로젝트입니다. 오가닉 맵의 지도 데이터의 주요 소스이며 위키백과와 유사하게 작동합니다. 장소를 추가하거나 편집하면 전 세계 수백만 명의 사용자가 사용할 수 있습니다.\n커뮤니티에 참여하여 모두를 위한 더 나은 지도를 만드는 데 도움을 주세요!
+    lt = OpenStreetMap.org (OSM) - tai bendruomenės projektas, kuriuo siekiama sukurti nemokamą ir atvirą žemėlapį. Tai pagrindinis žemėlapių duomenų šaltinis "Organiniuose žemėlapiuose" ir veikia panašiai kaip "Vikipedija". Galite pridėti ar redaguoti vietoves, ir jos tampa prieinamos milijonams naudotojų visame pasaulyje.\nPrisijunkite prie bendruomenės ir padėkite kurti geresnį žemėlapį visiems!
+    lv = OpenStreetMap.org (OSM) ir kopienas projekts, kura mērķis ir izveidot bezmaksas un atvērtu karti. Tas ir galvenais karšu datu avots Organic Maps un darbojas līdzīgi kā Wikipedia. Jūs varat pievienot vai rediģēt vietas, un tās kļūst pieejamas miljoniem lietotāju visā pasaulē.\npievienojieties kopienai un palīdziet izveidot labāku karti visiem!
```

Comment:
OpenStreetMap.org (OSM) ir kopienas projekts, kura mērķis ir izveidot atvērtajos datos balstītu karti, kura ir pieejama bez maksas. Tas ir Organic Maps galvenais karšu datu avots un darbojas līdzīgi kā Vikipēdija. Jūs varat pievienot vai rediģēt vietas, un tās kļūst pieejamas miljoniem lietotāju visā pasaulē.\nPievienojieties kopienai, lai palīdzētu izveidot labāku karti visiem!